src/factory/client.ts

```diff
@@ -436,21 +438,34 @@ export const createClient = (
   network: Network,
   mnemonic: string,
   accountType: AccountType,
-  derivationPath: string
+  derivationPath: string,
+  publicKey?: string,
```

Comment:
better put these 2 params into one object:

```
derivationPath: string,
bitcoinLedgerCache?: { publicKey: string, chainCode: string }
```

This makes the purpose of these options much clearer

Reply:
done